features/airgapped.feature

```diff
@@ -47,3 +47,53 @@ Feature: Performing attach using ua-airgapped
         Examples: ubuntu release
             | release | machine_type  |
             | jammy   | lxd-container |
+
+    Scenario Outline: airgapped environment with same apt url for different services
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        # set up the apt mirror configuration
+        Given a `jammy` `<machine_type>` machine named `mirror`
+        When I run `add-apt-repository ppa:yellow/ua-airgapped -y` `with sudo` on the `mirror` machine
+        And I run `apt-get update` `with sudo` on the `mirror` machine
+        And I run `apt-get install apt-mirror get-resource-tokens ua-airgapped -yq` `with sudo` on the `mirror` machine
+        And I download the service credentials on the `mirror` machine
+        And I extract the `esm-infra` credentials from the `mirror` machine
+        And I extract the `esm-apps` credentials from the `mirror` machine
+        And I set the apt-mirror file for `<release>` with the `esm-infra,esm-apps` credentials on the `mirror` machine
```

Comment:
Does apt-mirror on this line need to be decorated like `apt-mirror` on the next line?